### DISCUSSION TITLE
Daughter's wedding rehearsal dinner & general questions

### DISCUSSION THREAD

**Discussion by GaryH-hero12** [gopro/labs#978] [Q&A]
I just joined this discussion group because I recently bought (last week) my first GoPro (Hero 12). I have been reading and watching videos to learn about the camera. I am planning to use GoPro lab’s Firmware so that I can quickly change settings for different activities (e.g., vacations). Below are my questions. Please remember that I am a GoPro newbie so please be clear with responses. Also, I have also been trying to learn free version of DaVinci Resolve for some easy edits (e.g., Clarity VX to remove background noise) and put different clips together. 

1)	**Daughter’s wedding rehearsal:** I want to use camera at the wedding rehearsal. It is at a restaurant and I plan to use a tripod. It is indoors. Which settings do you suggest?

2)	**General DAYTIME settings for walking around**: Can you suggest the best, and most flexible, settings for walking around during the day. No special action shots. 

3)	**General NIGHTTIME settings for walking around**: Can you suggest the best, and most flexible, settings for walking around during the evening (streetlights). No special action shots. I assume this is different from the restaurant version (wedding rehearsal above).

4)	**General LABS questions:** In the videos I watched, people suggested I change certain settings permanently (EV Comp= -0.5, Sharpness=low, Color=Natural, Wind=Auto). If I use GoPro labs firmware, do I need to change these for every situation or is there a way general change for all activities (i.e., permanently unless otherwise changed)?

Thank you in advance.

**Comment by dnewman-gpsw** [comment]
Labs answers here, (I'll leave the other question to others), or post this on reddit.com/r/gopro

4. These are just preset settings, Labs will use them just like a stock camera. Just have them set for each preset you intend to use.  I also agree with these settings.

**Comment by GaryH-hero12** [comment]
Thank you.


-------------------------------------------------

Gary A. Heiman, PhD

Vice Chair and Undergraduate Director

Professor, Department of Genetics

Rutgers University

Nelson Biology Laboratories | Room B412

604 Allison Rd, Piscataway, NJ  08854

P) 848-445-9576

F) 732-445-1147

Email) ***@***.******@***.***>



CONFIDENTIALITY NOTICE: This message is intended only for designated addressee(s), and may contain information that is privileged, confidential, or exempt from disclosure under HIPAA or applicable law. If you are not an intended recipient, you may not copy, use, act upon, or disseminate any part(s) of this message. If received in error, please notify the original sender immediately and purge all relevant information.

________________________________
